packages/pricing/src/models/rule-type.ts

```diff
@@ -0,0 +1,34 @@
+import { generateEntityId } from "@medusajs/utils"
+import {
+  BeforeCreate,
+  Entity,
+  OptionalProps,
+  PrimaryKey,
+  Property,
+} from "@mikro-orm/core"
+
+type OptionalFields = "default_priority"
+
+@Entity()
+class RuleType {
+  [OptionalProps]?: OptionalFields
+
+  @PrimaryKey({ columnType: "text" })
+  id!: string
+
+  @Property({ columnType: "text" })
+  name: string
+
+  @Property({ columnType: "text", index: "IDX_rule_type_key_value" })
+  key_value: string
```

Comment:
thought: Can we come up with a more descriptive column name? 

Couple of ideas:
- `rule_property`
- `rule_identifier`
- `rule_attribute`

What do you think? 

cc @carlos-r-l-rodrigues

Reply:
I think `rule_attribute` or `rule_property` are good options here, could be either 😄

Reply:
going for rule_attribute